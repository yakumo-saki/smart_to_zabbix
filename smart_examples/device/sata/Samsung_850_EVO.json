{
    "json_format_version": [
      1,
      0
    ],
    "smartctl": {
      "version": [
        7,
        2
      ],
      "svn_revision": "5155",
      "platform_info": "x86_64-linux-5.13.19-5-pve",
      "build_info": "(local build)",
      "argv": [
        "smartctl",
        "-a",
        "-j",
        "/dev/sdd"
      ],
      "exit_status": 0
    },
    "device": {
      "name": "/dev/sdd",
      "info_name": "/dev/sdd [SAT]",
      "type": "sat",
      "protocol": "ATA"
    },
    "model_family": "Samsung based SSDs",
    "model_name": "Samsung SSD 850 EVO 250GB",
    "serial_number": "S3*************",
    "wwn": {
      "naa": 5,
      "oui": 9528,
      "id": 56951657723
    },
    "firmware_version": "EMT03B6Q",
    "user_capacity": {
      "blocks": 488397168,
      "bytes": 250059350016
    },
    "logical_block_size": 512,
    "physical_block_size": 512,
    "rotation_rate": 0,
    "form_factor": {
      "ata_value": 3,
      "name": "2.5 inches"
    },
    "trim": {
      "supported": true,
      "deterministic": false,
      "zeroed": false
    },
    "in_smartctl_database": true,
    "ata_version": {
      "string": "ACS-2, ATA8-ACS T13/1699-D revision 4c",
      "major_value": 1020,
      "minor_value": 57
    },
    "sata_version": {
      "string": "SATA 3.1",
      "value": 127
    },
    "interface_speed": {
      "max": {
        "sata_value": 14,
        "string": "6.0 Gb/s",
        "units_per_second": 60,
        "bits_per_unit": 100000000
      },
      "current": {
        "sata_value": 3,
        "string": "6.0 Gb/s",
        "units_per_second": 60,
        "bits_per_unit": 100000000
      }
    },
    "local_time": {
      "time_t": 1646990350,
      "asctime": "Fri Mar 11 18:19:10 2022 JST"
    },
    "smart_status": {
      "passed": true
    },
    "ata_smart_data": {
      "offline_data_collection": {
        "status": {
          "value": 0,
          "string": "was never started"
        },
        "completion_seconds": 0
      },
      "self_test": {
        "status": {
          "value": 0,
          "string": "completed without error",
          "passed": true
        },
        "polling_minutes": {
          "short": 2,
          "extended": 133
        }
      },
      "capabilities": {
        "values": [
          83,
          3
        ],
        "exec_offline_immediate_supported": true,
        "offline_is_aborted_upon_new_cmd": false,
        "offline_surface_scan_supported": false,
        "self_tests_supported": true,
        "conveyance_self_test_supported": false,
        "selective_self_test_supported": true,
        "attribute_autosave_enabled": true,
        "error_logging_supported": true,
        "gp_logging_supported": true
      }
    },
    "ata_sct_capabilities": {
      "value": 61,
      "error_recovery_control_supported": true,
      "feature_control_supported": true,
      "data_table_supported": true
    },
    "ata_smart_attributes": {
      "revision": 1,
      "table": [
        {
          "id": 5,
          "name": "Reallocated_Sector_Ct",
          "value": 100,
          "worst": 100,
          "thresh": 10,
          "when_failed": "",
          "flags": {
            "value": 51,
            "string": "PO--CK ",
            "prefailure": true,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 9,
          "name": "Power_On_Hours",
          "value": 94,
          "worst": 94,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 29376,
            "string": "29376"
          }
        },
        {
          "id": 12,
          "name": "Power_Cycle_Count",
          "value": 99,
          "worst": 99,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 300,
            "string": "300"
          }
        },
        {
          "id": 177,
          "name": "Wear_Leveling_Count",
          "value": 51,
          "worst": 51,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 19,
            "string": "PO--C- ",
            "prefailure": true,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": false
          },
          "raw": {
            "value": 1041,
            "string": "1041"
          }
        },
        {
          "id": 179,
          "name": "Used_Rsvd_Blk_Cnt_Tot",
          "value": 100,
          "worst": 100,
          "thresh": 10,
          "when_failed": "",
          "flags": {
            "value": 19,
            "string": "PO--C- ",
            "prefailure": true,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": false
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 181,
          "name": "Program_Fail_Cnt_Total",
          "value": 100,
          "worst": 100,
          "thresh": 10,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 182,
          "name": "Erase_Fail_Count_Total",
          "value": 100,
          "worst": 100,
          "thresh": 10,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 183,
          "name": "Runtime_Bad_Block",
          "value": 100,
          "worst": 100,
          "thresh": 10,
          "when_failed": "",
          "flags": {
            "value": 19,
            "string": "PO--C- ",
            "prefailure": true,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": false
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 187,
          "name": "Uncorrectable_Error_Cnt",
          "value": 100,
          "worst": 100,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 190,
          "name": "Airflow_Temperature_Cel",
          "value": 68,
          "worst": 43,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 32,
            "string": "32"
          }
        },
        {
          "id": 195,
          "name": "ECC_Error_Rate",
          "value": 200,
          "worst": 200,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 26,
            "string": "-O-RC- ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": true,
            "event_count": true,
            "auto_keep": false
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 199,
          "name": "CRC_Error_Count",
          "value": 99,
          "worst": 99,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 62,
            "string": "-OSRCK ",
            "prefailure": false,
            "updated_online": true,
            "performance": true,
            "error_rate": true,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 979,
            "string": "979"
          }
        },
        {
          "id": 235,
          "name": "POR_Recovery_Count",
          "value": 99,
          "worst": 99,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 18,
            "string": "-O--C- ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": false
          },
          "raw": {
            "value": 84,
            "string": "84"
          }
        },
        {
          "id": 241,
          "name": "Total_LBAs_Written",
          "value": 99,
          "worst": 99,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 120687766502,
            "string": "120687766502"
          }
        }
      ]
    },
    "power_on_time": {
      "hours": 29376
    },
    "power_cycle_count": 300,
    "temperature": {
      "current": 32
    },
    "ata_smart_error_log": {
      "summary": {
        "revision": 1,
        "count": 0
      }
    },
    "ata_smart_self_test_log": {
      "standard": {
        "revision": 1,
        "count": 0
      }
    },
    "ata_smart_selective_self_test_log": {
      "revision": 1,
      "table": [
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        },
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        },
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        },
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        },
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        }
      ],
      "current_read_scan": {
        "lba_min": 0,
        "lba_max": 65535,
        "status": {
          "value": 0,
          "string": "was never started"
        }
      },
      "flags": {
        "value": 0,
        "remainder_scan_enabled": false
      },
      "power_up_scan_resume_minutes": 0
    }
  }
  