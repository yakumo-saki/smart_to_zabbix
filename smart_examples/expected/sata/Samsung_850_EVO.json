{
    "model": "Samsung SSD 850 EVO 250GB",
    "type": "sat",
    "rotation_rate": 0,
    "protocol": "ATA",
    "serial_number": "S3*************",
    "power_cycle": 300,
    "power_on_hours": 29376,
    "temperature": 32,
    "ssd.bytes_written": 61792136449024,
    "ssd.bytes_written_max": 75000000000000,
    "ssd.lifespan": null,
    "smart_status_passed": 1
}